lib/github.atd

```diff
@@ -786,7 +786,7 @@ type bool_as_string = string wrap <ocaml
 type web_hook_config = {
   url: string;
   content_type: string;
-  insecure_ssl: bool_as_string;
+  ~insecure_ssl <ocaml default="true">: bool_as_string;
```

Comment:
convenient for self-signed things, but I guess we should go letsencrypt/acme instead...